ngraph/core/shape_inference/include/tile_shape_inference.hpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#pragma once
+#include <openvino/op/tile.hpp>
+
+#include "shape_infer_utils.hpp"
+#include "utils.hpp"
+namespace ov {
+namespace op {
+namespace v0 {
+
+template <class T>
+void shape_infer(const Tile* op,
+                 const std::vector<T>& input_shapes,
+                 std::vector<T>& output_shapes,
+                 const std::map<size_t, std::shared_ptr<ngraph::runtime::HostTensor>>& constant_data = {}) {
+    NODE_VALIDATION_CHECK(op, input_shapes.size() == 2 && output_shapes.size() == 1);
+    const auto& arg_shape = input_shapes[0];
+    auto& repeats_shape = input_shapes[1];
+    auto& output_shape = output_shapes[0];
+    using DimType = typename std::iterator_traits<typename T::iterator>::value_type;
+    std::vector<int64_t> axes_val;
+    NODE_VALIDATION_CHECK(op, repeats_shape.rank().compatible(1), "PartialShape of repeats must be of rank 1");
+
+    //Get repeats
+    bool axes_are_known = get_data_as_int64<T>(1, op, axes_val, constant_data);
+    const auto arg_rank = arg_shape.rank();
+    if (arg_rank.is_static() && (axes_are_known || repeats_shape[0].is_static())) {
```

Comment:
we never checked repeats_shape has static rank. please add tests in type_prop for that

Reply:
Hi jane, this is where I fix the tile OnnxBackendLayerTest, but I could still add a Ngraph tests for this situation

Reply:
fix in 4a44503e12d519ae51cc7989ffc24bce78c037a7